src/circuits/joinsplit.tcc

```diff
@@ -399,6 +399,14 @@ public:
         }
     }
 
+    // Check the booleaness of packing variables
+    // Check the booleanees of phi and the a_sks
+    // Check value 0
```

Comment:
`Check value of ZERO` -> `Check value of ZERO (i.e. that ZERO = FieldT::zero())`

Reply:
Fixed